src/aggregator/client/options.go

```diff
@@ -58,6 +58,12 @@ const (
 	// before it must wait for an existing batch to be flushed to an instance.
 	defaultInstanceQueueSize = 128
 
+	// defaultInstanceMaxQueueSizeBytes determines how many bytes across all payloads
+	// can be buffered in the queue before the eviction policy kicks in (policy is
+	// specified in DropType)
+	// make it similar to defaultMaxBatchSize for starters.
```

Comment:
mark maxBatchSize as deprecated and remove this comment? encoder batching no longer applies

Reply:
cc @vdarulis -- do you feel good about this default here? 

Current behavior as of this diff (iiuc--@shaan420 correct me if I'm wrong):

- Encoders encode until they reach `MaxBatchSize()` (http://github.com/m3db/m3/blob/3a896841fd094287b472415aa6b92fd7f914f9c7/src/aggregator/client/writer.go#L246-L246) (same as master)
- When they reach max batch size, they're placed on the queue.  (same as master)
- The queue enqueues data until one of `InstanceQueueSize` or `InstanceMaxQueueSizeBytes` is reached.

I've unfortunately forgotten why we switched to the bytes based limit... apologies on that. Remind me @shaan420 ?